crates/hpos_connect_hc/src/hf_agent.rs

```diff
@@ -10,6 +10,7 @@ use std::sync::Arc;
 /// Struct giving access to local instance of HHA on HPOS
 /// `config` of type `holo_config::Config` represents CLI params and can be passed
 /// to describe local running environment
+#[derive(Clone)]
```

Comment:
can we remove these clonable features, we should not have clones of these connection because that could cause issues if someone is not careful

Reply:
I removed it for the HfAgent.   CoreAppAgent is currently in use in our services.